vue-app/src/views/TransactionSuccess.vue

```diff
@@ -0,0 +1,211 @@
+<template>
+  <div class="container">
+    <div class="image-wrapper">
+      <img class="docking" src="@/assets/docking.png" />
+      <img class="money" src="@/assets/money.gif" />
+      <img class="money" src="@/assets/confetti.gif" />
+    </div>
+    <div class="content">
+      <span class="contributed-icon">🎉</span>
+      <p
+        v-if="$route.params.type === 'reallocation'"
+        class="contributed-header"
+      >
+        {{ formatContribution() }}
+        {{ $store.state.currentRound.nativeTokenSymbol }} reallocated!
+      </p>
+      <p
+        v-else-if="$route.params.type === 'contribution'"
+        class="contributed-header"
+      >
+        You just contributed!
+      </p>
+      <div>
+        <p
+          v-if="$route.params.type === 'reallocation'"
+          class="contributed-content"
+        >
+          Your choices have been updated! You can update your choices again any
+          time in the next
+          <span class="contributed-content-bold"
+            >{{ reallocationTimeLeft.days }} days
+            {{ reallocationTimeLeft.hours }} hours</span
+          >.
+        </p>
+        <p
+          v-else-if="$route.params.type === 'contribution'"
+          class="contributed-content"
+        >
+          Thanks for contributing to the Eth2 ecosystem. If you change your
+          mind, you have
+          <span class="contributed-content-bold"
+            >{{ reallocationTimeLeft.days }} days
+            {{ reallocationTimeLeft.hours }} hours</span
+          >
+          to reallocate your contributions.
+        </p>
+        <div class="receipt">
+          <transaction-receipt :hash="`${$route.params.hash}`" />
+        </div>
+        <div class="input-button">
+          <button class="contributed-button" @click="redirectToProjects()">
+            <span>OK</span>
+          </button>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+// Libraries
+import { BigNumber } from 'ethers'
+import Vue from 'vue'
+import Component from 'vue-class-component'
+
+// API
+import { RoundInfo, TimeLeft } from '@/api/round'
+
+// Components
+import TransactionReceipt from '@/components/TransactionReceipt.vue'
+
+// Utils
+import { formatAmount } from '@/utils/amounts'
+import { getTimeLeft } from '@/utils/dates'
+
+@Component({
+  components: { TransactionReceipt },
+})
+export default class TransactionSuccess extends Vue {
+  get contribution(): BigNumber | null {
+    return this.$store.state.contribution
+  }
+
+  get currentRound(): RoundInfo | null {
+    return this.$store.state.currentRound
+  }
+
+  get reallocationTimeLeft(): TimeLeft | string {
+    return this.currentRound
```

Comment:
Regarding the `currentRound` check:
I think it is a valid check but it is in the incorrect place. I think it should be a higher level check we should be doing in a different part of the app (perhaps we could do something in the router level where we could require specific state variables to display the view). 
I've seen this checking, in other views as well. IMO, if no `currentRound` exists, we shouldn't display anything and display an error view.
But this is out of the scope of this PR, I just wanted to throw my thoughts about it.

**In the context of this PR**, I would just add this check on a higher element in the tree and avoid doing that check in the rest of the code:
```
<template>
  <div class="container">
    <div v-if="!currentRound">
      // error message view
    </div>

    <div v-if="currentRound">
      // happy path
    </div>
  </div>
</template>
```

Reply:
Yeah I was trying to use the `get currentRound` information in this get to avoid the typescript errors I was getting, but I should have just gone right to the source for this to avoid it. Changing to just use the store value here instead to avoid this typing confusion. Thanks for the review!